src/components/shared/Footer.tsx

```diff
@@ -7,12 +7,7 @@ import {pageLinks} from "@site/src/constants/routes"
 const Footer = (): JSX.Element => {
   const year = new Date().getFullYear()
   return (
-    <footer className="bg-tailCall-dark-700 pt-SPACE_10 pb-SPACE_05 flex flex-col items-center justify-center gap-SPACE_10 w-full relative px-SPACE_04">
-      <img
-        src={require("@site/static/images/about/grid-dark.png").default}
-        alt="grid background"
-        className="absolute inset-0 w-full h-full"
-      />
+    <footer className="bg-[#1C1D1F] grid-background pt-SPACE_10 pb-SPACE_05 flex flex-col items-center justify-center gap-SPACE_10 w-full relative px-SPACE_04">
```

Comment:
This grid a bit different :)

Reply:
Check the design https://www.figma.com/design/mMLCy6XZinopm6QrEqOovP/Tailcall-Website-Redesign---Shared?node-id=2913-221753&t=ImOZMgbhPu8a9kMe-0

Reply:
Updated!